digdag-docs/src/operators/s3_wait.md

```diff
@@ -81,6 +81,21 @@ For more information about SSE-C, See the [AWS S3 Documentation](http://docs.aws
   An optional flag to control whether to use path-style or virtual hosted-style access when accessing S3.
   *Note:* Enabling `path_style_access` also requires specifying a `region`.
 
+* **timeout**: TIMEOUT
+
+  Set timeout.
+
+  Examples: wait 120 seconds
+
+  ```
+  timeout: 120s
+  ```
+
+* **ignore_timeout_error**: true/false (default:false)
```

Comment:
What is the use case of setting `true` here? Depending on it, shouldn't s3_wait set a store_param when timeout is ignored so that following tasks can know whether the file exists or not? Some workflows may want to branch using `if> ${store_parameter_set_by_s3_wait}`?
Please consider naming together. An example is `continue_on_timeout` but please think.

Reply:
And AbstractWaitOperatorFactory (which is a base class of td_wait and td_wait_table).

Reply:
The reason I introduced this parameter is demand from internal.
But I will check it in detail.

Reply:
Better understanding of users is very good to write better. If you have more public channels to get information about users and use cases, you are good OSS developer :-)

Reply:
@frsyuki I would like to introduce s3.last_object.stored : true/false.

I investigated the option name for this behavior. But not good name found.
In Airflow, sensor has _soft_fail_ parameter but downstream tasks will be skipped.
In GitHub Actions, they have _jobs.<job_id>.steps.continue-on-error_ .
It is near to _continue_on_timeout_ which you proposed as example.
So I would like to use the name _continue_on_timeout_